roles/ceph-dashboard/tasks/configure_dashboard_backends.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: get current mgr backend - ipv4
+  set_fact:
+      mgr_server_addr: "{{ hostvars[dashboard_backend]['ansible_all_ipv4_addresses'] | ips_in_ranges(public_network.split(',')) | first }}"
+  when:
+    - ip_version == 'ipv4'
```

Comment:
If there's only one condition then you don't have to put it on a dedicated line

Reply:
+1, Done